handbook/tools/deployment.md

```diff
@@ -29,6 +29,6 @@ We recommend using [Netlify's file-based configuration](https://docs.netlify.com
 
 ### Heroku
 
-:: warning
-TODO [#110](https://github.com/ubclaunchpad/docs/issues/110)
-::
+[Heroku](https://www.heroku.com/what) is a cloud platform that allows you deploy a variety of applications. Heroku can be used as a [remote target](https://devcenter.heroku.com/articles/git), or you can have it [track a branch on a Github repository](https://devcenter.heroku.com/articles/github-integration) and auto-deploy whenever that branch is updated. You can find more information on [Heroku's documentation](https://devcenter.heroku.com/).
```

Comment:
For general Heroku info, can you link to the resources section instead of having a copy? (sorry this wasn't very clear!)

Reply:
This page in handbook is meant to detail Launch Pad -specific advice (for example, if we have a sponsored account or similar)

In this case, we could point out other projects that have deployed with heroku (sync?) and maybe what the free option is good for

Reply:
Aaaah of course, a quick look into `inertia` would've given me that info 🤦 Anywhoo done!